### PR TITLE
v2.0.0 – Updated Menulog colour palette (and the way it can be imported)

### PR DESCRIPTION
**Pull request by ashleynolan**
### Changed
- Menulog theming updated (breaking change).
  The Menulog colours are now specified as a mixin that is called as part of individual modules if the $theme variable is set to 'ml'.  This is so that each project importing fozzie modules only has to specify the theme variable once and each module will take care of it's own colour/variable overrides.
- Menulog Colour Scheme has been updated in line with the global UI palette provided by AU design team.

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines
- UI Documentation will be created once Menulog theming has been done (separate task)
- [x] This code has been checked with regard to our accessibility standards
  - [x] The colours added/changed have been colour contrast tested [[link]](http://webaim.org/resources/contrastchecker/)

### CODE REVIEW

File: src/scss/menulog.scss [hunk 1]
```diff
@@ -1,90 +1,43 @@
 // =================================
-// Menulog Color palette
+// Menulog Color Scheme
```
Comment (DamianMullins) [inline src/scss/menulog.scss]: Can this file be a partial now?

Comment (ashleynolan) [reply src/scss/menulog.scss]: Do you mean, can it be in a folder called partials?

I could put it in a folder called themes, as that's kinda what it is I guess?

Comment (DamianMullins) [reply src/scss/menulog.scss]: It was more of a question really; as the filename doesn't start with an underscore it implies, to me, that this file would be used generate a separate CSS file. Is that correct?

Comment (ashleynolan) [reply src/scss/menulog.scss]: Ah ok – makes sense. Yeh, should be able to be a file partial now – will update now

Comment (ashleynolan) [reply src/scss/menulog.scss]: Just updated that now 👍

**Comment by ashleynolan** [comment]
@bulutcy This will be integrated automatically with v1 of Fozzie (which is on it's way), but if you need help integrating before then just give me a shout

**Pull request merged**